cmake/Modules/FindCUDNN.cmake

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
Are you sure that this file is windows compatible?

Reply:
Does the CI not test windows Cuda build?

Reply:
But yes, this file is very much equivalent to what we had before. It only changes the way we notify the user and it adds a default path.
Windows users may need to overwrite the default path. But nothing that worked prior to this PR should stop working afterwards.